app/manager/models.py

```diff
@@ -108,19 +108,26 @@ def __str__(self):
         return self.name
 
 
+class VMSettings(models.Model):
```

Comment:
I think hardware is better for the name of this class as this is not only a vm but a bare metal server

Reply:
Done :D